docs/src/Tooltip.doc.js

```diff
@@ -1,165 +1,822 @@
+              />
+            </Box>
+            <Box paddingY={1}>
+              <RadioButton
+                checked={claimed === "other"}
+                id="other"
+                label="Other pins"
+                name="claimed"
+                onChange={() => setClaimed( "other" )}
+                value="other"
+              />
+            </Box>
+          </Box>
+        </Flex>
+        <Flex
+          direction="column"
+          gap={2}
+        >
+          <Flex
+            alignItems="center"
+            gap={1}
+          >
+            <Text
+              weight="bold"
+              size="lg"
+            >
+              Device
+            </Text>
+            <Tooltip
+              idealDirection="right"
+              text="See stats for the different devices your Pinterest traffic is coming from.">
+              <Icon
+                icon="info-circle"
+                accessibilityLabel="Information"
+                color="darkGray"
+              />
+            </Tooltip>
+          </Flex>
+          <Box
+            display="flex"
+            direction="column"
+          >
+          <Box paddingY={1}>
+            <RadioButton
+              checked={device === "all"}
+              id="alldevices"
+              label="All"
+              name="device"
+              onChange={() => setDevice( "all" )}
+              value="all"
+            />
+            </Box>
+            <Box paddingY={1}>
+              <RadioButton
+                checked={device === "mobile"}
+                id="mobile"
+                label="Mobile"
+                name="device"
+                onChange={() => setDevice( "mobile" )}
+                value="mobile"
+              />
+            </Box>
+            <Box paddingY={1}>
+              <RadioButton
+                checked={device === "desktop"}
+                id="desktop"
+                label="Desktop"
+                name="device"
+                onChange={() => setDevice( "desktop" )}
+                value="desktop"
+              />
+            </Box>
+             <Box paddingY={1}>
+              <RadioButton
+                checked={device === "tablet"}
+                id="tablet"
+                label="Tablet"
+                name="device"
+                onChange={() => setDevice( "tablet" )}
+                value="tablet"
+              />
+            </Box>
+          </Box>
+        </Flex>
+      </Flex>
+    </ScrollableContainer>
+)}`}
+      />
+    </MainSection.Subsection>
+  </MainSection>,
 );
 
 card(
-  <Example
-    name="Accessibility"
+  <MainSection name="Writing">
+    <MainSection.Subsection>
+      <MainSection.Card
+        cardSize="md"
+        type="do"
+        description={`
+- Describe only the element that triggers Tooltip.
+- Focus on the action by beginning with a verb.
+- Use succinct and scannable language.
+- As much as possible, limit Tooltip to a maximum of 60 to 75 characters.
```

Comment:
"limit Tooltip's `text` to a maximum of…"

Reply:
DONE